src/app/services/map/map.service.ts

```diff
@@ -83,4 +85,8 @@ export class MapService {
     return this.http.get(`http://${environment.host}:${environment.port}/tweet/tweet-from-id`,
       {params: new HttpParams().set('tweet_id', id)});
   }
+
+  zoom(firePolygonLL) {
```

Comment:
not sure what does function `zoom` do.  it should serve as a communication function right? not actually doing zoom

Reply:
This zoom is not doing the zoomin, it serves as a communication function to inform the coremap to do the zoomin